### ISSUE TITLE
Optimization does not remove full unused var declaration

### ISSUE THREAD

**Issue by Zenwolf** [zynga/jasy#12]
Using Optimization does not remove a full unused var declaration assigned to an Object literal.

Jasy version: 0.4.6

Source Example:

``` javascript
__render: function () {

    var dimensions = {
        width: 100,
        height: 100
    };

    // other code removed that never references local dimensions var.
},
```

Optimization result:

``` javascript
__render:function(){{width:100,height:100}};
```

Js error:
`Uncaught SyntaxError: Unexpected Token :`

jasyscript.py code:

```
optimization = Optimization("variables", "declarations", "blocks")
formatting = Formatting()
```

**Comment by swernerx** [comment]
Thanks for reporting. This should be fixed since 0.5-beta5.

**Comment by Zenwolf** [comment]
Good to hear, thanks!
